apps/api/env/.env.unit.test

```
@@ -25,6 +25,6 @@ SQL_LOG_FORMAT=pretty
 DEPLOYMENT_ENV=test
 PROVIDER_PROXY_URL=http://localhost:3040
 AMPLITUDE_API_KEY=AMPLITUDE_API_KEY
-NOTIFICATIONS_API_BASE_URL=http://localhost:3081
+NOTIFICATIONS_API_BASE_URL=http://localhost:3090
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NOTIFICATIONS_API_BASE_URL key should go before the POSTGRES_DB_URI key

(UnorderedKey)